art/estimators/generation/tensorflow.py

```diff
@@ -100,8 +100,18 @@ def __init__(
             # TODO do the same thing for all not None variables
         self._sess = sess
 
-    def loss(self, z, image_adv):
-        pass
```

Comment:
@killianlevacher Did you need this method `loss(self, z, image_adv)` for something? Can we safely replace it with the new method below?